src/internal/observable/of.ts

```diff
@@ -5,6 +5,57 @@ import { empty } from './empty';
 import { scalar } from './scalar';
 import { Observable } from '../Observable';
 
+/**
+ * Converts the arguments to an observable sequence.
+ *
+ * <span class="informal">Each argument becomes a `next` notification.</span>
+ *
+ * ![](of.png)
```

Comment:
I realize I'm commenting on my own PR - but where do these files come from?

Reply:
Could you elaborate a bit more in detailed what's the problem? I actually don't get it sorry

Reply:
Sure - the images on the [docs site](https://rxjs-dev.firebaseapp.com/api/operators/mergeMap) don't seem to be loading:

![image](https://user-images.githubusercontent.com/5975076/42501052-c491f26e-842a-11e8-87ad-f7ece024e265.png)

![image](https://user-images.githubusercontent.com/5975076/42501113-e4fc5d46-842a-11e8-85e2-c65de4a5bf7d.png)

Is it still WIP?

Reply:
I am not sure when the last version was deployed, but there was a change that the images have to be generated. To do that you have to call `npm run tests2png` (there is a documentation about that, but I actually don't find it :D). Maybe this was forgotten last time, and it definetly has to be added to the deployment script. But you also have to do it locally. If you have any problems with that, just ping me

Reply:
@JWO719 Did you create an issue for this? Maybe this could be optional or at least ignored when fails. I don't know if you should be forced to install imagick if you only want to test eg. docs typos.